pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -1121,7 +1140,7 @@ func TestPriorityQueue_Update(t *testing.T) {
 				}
 
 				if obj, exists, _ := q.activeQ.Get(newQueuedPodInfoForLookup(newPod)); exists {
-					if tt.wantQ != activeQ {
+					if tt.wantQ != activeQ && tt.wantQ != notInAnyQueue {
```

Comment:
Most of this pr is copied from https://github.com/kubernetes/kubernetes/pull/119607, just made some minor adjustments.

Reply:
Why are we adding `&& tt.wantQ != notInAnyQueue` ?
If the test want the pod `notInAnyQueue` but found in activeQ, shouldn't the test fail?

Reply:
Sry, my bad. Misunderstood the logic.